db/migrate/20220914094309_create_orders.rb

```diff
@@ -0,0 +1,13 @@
+class CreateOrders < ActiveRecord::Migration[6.1]
+  def change
+    create_table :orders do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :address
+      t.float :total_amount
```

Comment:
Instead of having a column total_amount in order, we should add a column price in order_item. which will hold the price of the product at the time of order.